dev-libs/libucl/libucl-0.9.2.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY=" Universal configuration library parser"
+DESCRIPTION="Libucl is a parser and C API to parse and generate ucl objects."
+HOMEPAGE="https://github.com/vstakhov/libucl"
+COPYRIGHT="2013-2014, Vsevolod Stakhov"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/vstakhov/libucl/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="f63ddee1d7f5217cac4f9cdf72b9c5e8fe43cfe5725db13f1414b0d8a369bbe0"
+SOURCE_FILENAME="$portVersion.tar.gz"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="7.2.0"
+libVersionCompat="$libVersion compat >= 7"
+
+PROVIDES="
+	libucl$secondaryArchSuffix = $portVersion
+	lib:libucl$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libucl${secondaryArchSuffix}_devel = $portVersion
+	devel:libucl${secondaryArchSuffix} = $libVersion
+	"
+REQUIRES_devel="
+	libucl${secondaryArchSuffix} == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcurl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:cmake
+	cmd:curl_config$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sed
+	"
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	rm -f $libDir/libucl.a $libDir/libucl.la
+
+	# set up the develop directory correctly
```

Comment:
The description here is not fully accurate, what it does is symlinks the libraries to the develop package and moves the libucl.so there, so maybe you can drop this line?

Reply:
updated (removed)